### ISSUE TITLE
Missing key

### ISSUE THREAD

**Issue by PolyVinalDistillate** [circuitikz/circuitikz#99]
I'm fairly certain this must be an intentional change, but I recently update my LaTeX distribution and custom components I have making use of:
`\pgfkeysvalueof{/tikz/circuitikz/bipoles/length}`
no longer compile. I eventually tracked this to the removal of 
`\ctikzset{bipoles/length/.initial=1.4cm}`
which used to reside on line 62 of pgfcirc.defines.tex.
As a quick fix, I have added this to the start of my custom components file.

I suspect I'm grossly misusing this parameter as a length in my own components, but thought I'd flag it up. There are a few examples I've come across that use this key to set dimensions.

**Comment by PolyVinalDistillate** [comment]
I have looked through the changes made on GitHub, and in most places can simply replace
`\pgfkeysvalueof{/tikz/circuitikz/bipoles/length}`
with
`\pgf@circ@Rlen`
as seems to be the preferred method from your files.

Still a bit stumped on getting `\pgf@circ@Rlen` into a length register defined with `\newlength`, but I'm happy to explicitly define this as 1.4 cm for now! I'm still getting to grips with the intricacies of pgf stuff, but consider this issue closed as far as I'm concerned. Sorry if I wasted anyone's time!
